### PR TITLE
Update install instructions

### PR DESCRIPTION
**Pull request by artlovecode**
As per https://github.com/spyder-ide/spyder-vim/issues/33

### CODE REVIEW

**Comment by andfoy** [comment]
@audun- Thanks for your contribution, however, I would prefer to add these instructions under the ``Testing and development`` section, as packages will be released on the near future

File: README.md [hunk 1]
```diff
@@ -12,9 +12,11 @@ Spyder Plugin for executing Vim commands inside the code editor. We currently su
 ## Installation
 To install this plugin, you can use either ``pip`` or ``conda`` package managers, as it follows:
 
-Using pip:
+Build from source:
 ```
-pip install spyder-vim
+git clone https://github.com/spyder-ide/spyder-vim
+cd spyder-vim
+python setup.py install .
```
Comment (ccordoba12) [inline README.md]: I think this should be `pip install .` instead. @andfoy, what do you way?

Comment (andfoy) [reply README.md]: I always prefer using ``pip install .``. However, these instructions shouldn't be under the Installation section

Comment (ccordoba12) [reply README.md]: Yep, sure.

**Pull request closed without merging**